util/ibex_config.py

```diff
@@ -127,19 +127,82 @@ def get_config_dicts(config_file):
     return config_yaml
 
 
-def _config_dict_to_fusesoc_opts(config_dict):
-    fusesoc_cmd = []
-    for parameter, value in config_dict.items():
-        if isinstance(value, bool):
-            # For fusesoc boolean parameter are set to true if given on the
-            # command line otherwise false, it doesn't support an explicit
-            # --param=True style
-            if value:
-                fusesoc_cmd.append(shlex.quote('--' + parameter))
+class FusesocOpts:
+    def setup_args(self, arg_subparser):
+        output_argparser = arg_subparser.add_parser(
+            'fusesoc_opts', help=('Outputs options for fusesoc'))
+        output_argparser.set_defaults(output_fn=self.output)
+
+    def output(self, config_dict, args):
+        fusesoc_cmd = []
+        for parameter, value in config_dict.items():
+            if isinstance(value, bool):
+                # For fusesoc boolean parameter are set to true if given on the
+                # command line otherwise false, it doesn't support an explicit
+                # --param=True style
```

Comment:
Nit: "parameters", then the comma before "it" should be a full stop or semicolon.